exporter/clickhouselogsexporter/exporter.go

```diff
@@ -144,10 +144,10 @@ func attributesToSlice(attributes pcommon.Map, forceStringValues bool) (response
 			switch v.Type().String() {
 			case "INT":
 				response.IntKeys = append(response.IntKeys, formatKey(k))
-				response.IntValues = append(response.IntValues, v.IntVal())
+				response.IntValues = append(response.IntValues, v.Int())
 			case "DOUBLE":
```

Comment:
These case values have changed. Please check once. Also, don't use the string comparison instead, use the type itself.

Reply:
Aah thanks, missed it during testing. Have updated it.